### PR TITLE
Internal: add CI for Icon svg file validation

### PR DESCRIPTION
**Pull request by AlbertCarreras**
Follow up to [Icon: clean up icon svg files (#1401)](https://github.com/pinterest/gestalt/pull/1401)

Automating Icon svg file validation  with CI test.

FAILING EXAMPLE:
image

PASSING EXAMPLE:
image

### CODE REVIEW

File: scripts/validateIconSvgSpecs.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
```
Comment (AlbertCarreras) [inline scripts/validateIconSvgSpecs.js:1]: New CI test here

File: packages/gestalt/src/icons/hand-pointing.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...th d="M17.776,22.390 C13.775,23.978 9.246,22.359 7.219,18.770 C7.040,18.503 6.860,18.218 6.688,17.910 C5.132,15.163 3.028,13.782 2.588,13.581 C2.149,13.378 0.595,12.613 1.100,11.692 C1.381,11.180 2.163,10.801 2.878,10.726 C4.000,10.481 5.897,11.034 7.772,12.222 L4.054,3.419 C3.741,2.677 3.989,1.850 4.609,1.379 C4.612,1.376 4.616,1.373 4.621,1.369 C4.676,1.327 4.734,1.289 4.796,1.253 C4.810,1.247 4.821,1.239 4.835,1.231 C4.901,1.195 4.970,1.160 5.042,1.132 C5.044,1.131 5.044,1.131 5.045,1.131 C5.045,1.131 5.045,1.131 5.047,1.129 C5.119,1.101 5.193,1.079 5.267,1.060 C5.281,1.057 5.295,1.054 5.309,1.049 C5.380,1.035 5.449,1.021 5.518,1.013 C5.525,1.013 5.529,1.011 5.534,1.011 C6.317,0.928 7.093,1.347 7.406,2.088 L9.592,7.261 C9.643,6.623 10.055,6.035 10.709,5.776 C11.633,5.410 12.689,5.839 13.069,6.734 C13.071,6.742 13.074,6.752 13.077,6.759 C13.143,6.136 13.552,5.565 14.191,5.312 C15.117,4.946 16.173,5.373 16.551,6.271 C16.567,6.305 16.578,6.341 16.591,6.379 C16.680,5.784 17.079,5.249 17.695,5.006 C18.619,4.640 19.619,4.970 19.999,5.867 L20.001,5.872 C20.001,5.872 21.115,8.039 22.366,11.651 C22.413,11.791 22.457,11.931 22.495,12.072 C24.027,16.161 21.967,20.728 17.776,22.390"/></svg>
```
Comment (AlbertCarreras) [inline packages/gestalt/src/icons/hand-pointing.svg:1]: only change: order props and remove space at `" ><path`. it's not picking up changes

File: packages/gestalt/src/icons/history.svg [hunk 3]
```diff
@@ -1,2 +1 @@
...
-</svg>
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M11 5.75C10.31 5.75 9.75 6.31 9.75 7V11.518L12.866 14.634C13.11 14.878 13.43 15 13.75 15C14.07 15 14.39 14.878 14.634 14.634C15.122 14.146 15.122 13.354 14.634 12.866L12.25 10.482V7C12.25 6.31 11.69 5.75 11 5.75ZM22 11C22 17.065 17.065 22 11 22C5.561 22 0.993 18.107 0.138 12.744C0.007 11.926 0.565 11.157 1.383 11.026C2.202 10.896 2.97 11.453 3.101 12.271C3.722 16.17 7.044 19 11 19C15.411 19 19 15.411 19 11C19 6.589 15.411 3 11 3C8.782 3 6.704 3.941 5.216 5.5H6C6.829 5.5 7.5 6.171 7.5 7C7.5 7.829 6.829 8.5 6 8.5H0V3C0 2.171 0.671 1.5 1.5 1.5C2.329 1.5 3 2.171 3 3V3.465C5.049 1.299 7.932 0 11 0C17.065 0 22 4.935 22 11Z"/></svg>
```
Comment (AlbertCarreras) [inline packages/gestalt/src/icons/history.svg:1]: only change: order props and inline `</svg>`

File: yarn.lock [hunk 4]
```diff
@@ -5642,30 +5642,10 @@ caniuse-api@^3.0.0:
...
-caniuse-lite@^1.0.30001125:
-  version "1.0.30001154"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001154.tgz#f3bbc245ce55e4c1cd20fa731b097880181a7f17"
-  integrity sha512-y9DvdSti8NnYB9Be92ddMZQrcOe04kcQtcxtBx4NkB04+qZ+JUWotnXBJTmxlKudhxNTQ3RRknMwNU2YQl/Org==
-
-caniuse-lite@^1.0.30001146:
-  version "1.0.30001146"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001146.tgz#c61fcb1474520c1462913689201fb292ba6f447c"
-  integrity sha512-VAy5RHDfTJhpxnDdp2n40GPPLp3KqNrXz1QqFv4J64HvArKs8nuNMOWkB3ICOaBTU/Aj4rYAo/ytdQDDFF/Pug==
+caniuse-lite@^1.0.0, caniuse-lite@^1.0.30000792, caniuse-lite@^1.0.30000805, caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001038, caniuse-lite@^1.0.30001039, caniuse-lite@^1.0.30001043, caniuse-lite@^1.0.30001109, caniuse-lite@^1.0.30001125, caniuse-lite@^1.0.30001146:
```
Comment (AlbertCarreras) [inline yarn.lock:5645]: I have to remove this change

**Review by dangerismycat — APPROVED** [review]
LGTM!

**Pull request merged**